### ISSUE TITLE
Do not build external tools by default

### ISSUE THREAD

**Issue by JeremyMcCormick** [JeffersonLab/hps-mc#147]
External tools such as hps-java should not be built by default. Instead their locations should be specified using a config file. This depends on #146 being implemented, first.

**Comment by JeremyMcCormick** [comment]
Resolved by PR #151 (no separate issue branch)
